custom_components/foxess_modbus/entities/modbus_integration_sensor.py

```diff
@@ -24,7 +24,8 @@
 class ModbusIntegrationSensorDescription(SensorEntityDescription, EntityFactory):
     """Custom sensor description"""
 
-    address: int
+    # Unused for this sensor type
+    address = 0
```

Comment:
I think we can probably remove the field entirely?

Reply:
Unfortunately not...

```
    def update_callback(self, changed_addresses: set[int]) -> None:
        """Schedule a state update."""
        if self.entity_description.address in changed_addresses:
```

Reply:
Ah, that might be worth improving at some point.